features/node/proxy.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Node proxy configuration tests
+
+  # @author jhou@redhat.com
+  @admin
+  Scenario Outline: Proxy config should be applied to kubelet and crio
+    Given I switch to cluster admin pseudo user
+    When I run the :get client command with:
```

Comment:
@jhou1 try to avoid calling `:get` directly if possible.  It should be easy to add the class...I did it for you here https://github.com/openshift/verification-tests/pull/920.  After that's merged, then you can replace lines `7-12` with ```And the evalution of `proxy('cluster').raw_resource['spec']` is stored in the :proxy clipboard```

Reply:
Thanks @pruan-rht , I will replace it after this PR merges. Meanwhile, would you please enlighten me why we should avoid calling `:get` if possible?

Reply:
@jhou1 sure thing... The main reason we want to avoid using the raw `:get` is to adheres to the OOP paradigm.  By using encapsulation, we can address possible changes in the future easier.  In fact, that's one of the top comments that I've seen from reviewing PRs.  I can give a quick demo to the team or write a mojo about it...perhaps I should...

Reply:
Thank you @pruan-rht. I will follow the best practice.